lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -130,7 +130,7 @@ module.exports = [
               },
               {
                 resourceType: 'other',
-                countOverBudget: '1 request',
```

Comment:
should we change other budget to 0 to preserve this result? IMO we already exercise the different possible permutations enough but we thought we needed to set an other budget initially so... :)

Reply:
i considered that, but what a strange budget ...